package.json

```diff
@@ -89,6 +90,7 @@
     "react-native-push-notification": "^8.1.1",
     "react-native-qrcode-svg": "^6.0.6",
     "react-native-reanimated": "^2.2.3",
+    "react-native-restart": "daviroo/react-native-restart.git#master",
```

Comment:
Why do we need this fork?

Reply:
The released version of this wasn't working for me and there have been further changes on their master branch since.  I tested their main branch and it works but jest was unable to resolve the module without the lib directory which is generated as part of the build (not stored on github).  I branched the project and added the lib directory.